jet_tracking/jet_control.py

```diff
@@ -303,21 +381,28 @@ def jet_move_inline(injector, camera, params):
         # move the ROI to keep looking at the jet
         min_x = ROIx + (params.jet_x.get() / params.pxsize.get())
         camera.ROI.min_xyz.min_x.put(min_x)
-    # if params.state == [some state]
-    #     [use [x] for jet tracking]
-    # else if params.state == [some other state]:
-    #     [use [y] for jet tracking]
-    # else if params.state == [some other state]:
-    #     [revert to manual injector controls]
-    # etc...
-
-    # if jet is clear in image:
-    #     if jet_x != beam_x:
-    #         move injector.coarseX
-    #         walk_to_pixel(detector, motor, target) ??
-    # else if nozzle is clear in image:
-    #     if nozzleX != beam_x:
-    #         move injector.coarseX
-    # else:
-    #     if injector.coarseX.get() != beam_x:
-    #         move injector.coarseX
+
+
+def jet_scan(injector, cspad, params):
+  x_min = 0.0012
+  steps = 50
```

Comment:
TODO: Move magic numbers to function args or constants file

Reply:
Agreed that these should be moved outside the function block here.  Ideally these are stored in the "params" suite of PVs for jet tracking control

Reply:
🚀

Reply:
One more agree here.